nest/02.introduction.md

```diff
@@ -0,0 +1,51 @@
+
+Currently, to use Nest, one must use its Command Line Interface (CLI).
+This might sound intimidating for someone who has never seen a terminal or typed any command before, but if that's the case, don't worry! This guide is here to help you feel at home.
+
+## 1.2 Packages and dependencies
+It is common for a package to require other ones to work properly.
+For example, let's take a software that allows its users to send messages to each other, over the Internet.
+That software will need a way to send messages over the Internet.
+To do so, it will either implement its own solution, or use an existing library for that purpose.
+In the latter case, the software **requires** that library, because without it, the messages simply can't be sent.
+In other words, that library is a **dependency** of the software.
+
+Whenever you install a package via a package manager (and Nest as well), it will check if the package in question has any dependencies.
+If it does, Nest will install the dependencies that aren't already installed.
+So, when you install a package, don't be surprised if you see other packages getting installed too, they are just dependencies needed for that package to work properly.
+
+## 1.3 Repositories
+Packages are stored in repositories.
+Within each repository, there are a certain number of categories, and a package has to belong to one, and only one of them.
+The categories are a way to distinguish packages with the same name, but having different purposes.
+This means that we can have two packages with the same name, in the same repository but not in the same category, coexisting together, without any issue.
+
+`stable` is the repository enabled by default for Nest. It contains stable packages that are safe to use, and is maintained by the Raven-OS team.
+
+## 1.4 Package name syntax
+In Nest, the package names are structured in the following way: `repository::category/package#version`.
+If no version is specified, the latest one available is taken by default.
+
+However, most parts of this syntax may be omitted if it matches **one** and only **one** package.
+In a way, the only real mandatory part of this syntax is the package name itself. That means you can use:
+* `repository::package`, if that package is the only one with its name in the whole repository.
+* `category/package`, if the package's name is unique in this specific category, and if that category belongs to a single repository used by Nest.
+* `package`, if that package is the only one with its name amongst all the repositories used by Nest.
+
+If you're not sure on the repository, category, or name of a package, you can check on [Raven-OS official website](https://stable.raven-os.org).
+It lists all the packages in the repository `stable` of Nest, as well as the category they belong to, the latest version released, and a short description of what they provide.
+The repository `stable` is one of the official repositories available for Nest, and is enabled by default.
```

Comment:
This information was already given in the previous section. However not the exact same additional info was given.
Also  this means that the previous sentence ("If you're not sure on the repository, category, or name of a package...") is not entirely accurate, since the solution only works for packages within the `stable` repository.

Reply:
nice catch! I will remove the last sentence and remove `the repository` in the first one